build.gradle

```diff
@@ -216,7 +216,7 @@ pitest {
     ]
     threads = 10
     outputFormats = ['XML', 'HTML']
-    timestampedReports = true
```

Comment:
does this disable us getting times on the generated reports?

Reply:
no, with pitest it just puts it in a folder in the build artifacts like 202320101548, but we can't use wildcards in the publishHtml so it won't be able to store it in the Jenkins ui for easy access
with it set to false it just stores it in a reports folder instead

Reply:
ahh okay fair!